fc-community-backend/apps/cli/src/commands.ts

```diff
@@ -133,29 +140,34 @@ export const finaliseNotes = async (
     console.log(
       `finalising note on ${note.postUrl} created by ${note.creatorAddress}`,
     );
-    const finaliseTransactionResponse = await finaliseNoteHelper(note);
+    const [finalRating, finaliseTransactionResponse] =
+      await finaliseNoteHelper(note);
     responses.push(finaliseTransactionResponse);
     const mintTransactionResponse = await mintNote(
       note.content!,
       note.postUrl,
       note.creatorAddress,
+      finalRating,
     );
     responses.push(mintTransactionResponse);
   }
   console.log(responses);
+  console.log(`minted ${notesToFinalise.length} NFTs`);
   return responses;
 };
 
 export const mintNote = async (
   text: string,
   postUrl: string,
   creator: string,
+  finalRating: number,
```

Comment:
It feels like you could also pass a single object with all the fieds:
```js
{ test: string; postUrl: string; creator: string; finalRating: number }
```
This way you can just 
```
 mintNote(options)
 ``` 
 in your [CLI command parser ](https://github.com/factchain/factchain-community/pull/53/files#diff-ee3f95f82ca25236f75b955ea5783e3ac50a529e063a29e9a103c4a9067327a5L108)

Reply:
but that's a detail